src/Common/src/Interop/Windows/mincore/Interop.GetFullPathNameW.cs

```diff
@@ -11,27 +11,30 @@ partial class Interop
     partial class mincore
     {
         /// <summary>
-        /// WARNING: This overload does not implicitly handle long paths.
+        /// WARNING: This method does not implicitly handle long paths. Use GetFullPathName.
         /// </summary>
-        [DllImport(Libraries.CoreFile_L1, SetLastError = true, CharSet = CharSet.Unicode, BestFitMapping = false)]
-        internal unsafe static extern int GetFullPathNameW(char* path, int numBufferChars, char* buffer, IntPtr mustBeZero);
+        [DllImport(Libraries.CoreFile_L1, EntryPoint = "GetFullPathNameW", SetLastError = true, CharSet = CharSet.Unicode, BestFitMapping = false)]
+        internal unsafe static extern int GetFullPathNameUnsafe(char* path, int numBufferChars, char* buffer, IntPtr mustBeZero);
```

Comment:
There are lots of methods that use unsafe code / take ptr arguments that don't have Unsafe in the name.  More importantly, though, that's not the defining characteristic with regards to these various methods: the important thing we want to highlight is that it doesn't handle long paths / doesn't provide the wrapping behavior.  How about using "Core" as the suffix, to highlight that it's the core functionality, and then the other things wrap it?  Ideally we'd use it consistently instead of using "Private" in some places and other suffixes in others.  If we can't do that, I'd prefer something like "Native" here, since it at least that highlights that this is the Win32 P/Invoke function.